util/codecs/json.go

```diff
@@ -48,55 +49,44 @@ func LoadObjectFromFile(filename string, object interface{}) (err error) {
 	return
 }
 
+func writeBytes(writer io.Writer, object interface{}, prettyFormat bool) error {
+	var enc *json.Encoder
+	if prettyFormat {
+		enc = NewFormattedJSONEncoder(writer)
+	} else {
+		enc = json.NewEncoder(writer)
+	}
+	return enc.Encode(object)
+}
+
 // SaveObjectToFile implements the common pattern for saving an object to a file as json
 func SaveObjectToFile(filename string, object interface{}, prettyFormat bool) error {
```

Comment:
This is used in the Local object to write the local settings to disk (called from create command in goal). It is not covered by any tests downstream or here, which makes it slightly unnerving to change.